packages/synapse-react-client/src/components/SynapseSearchPageResults/SynapseSearchPageResults.tsx

```diff
@@ -2,18 +2,29 @@ import { Box, TextField, InputAdornment, Button } from '@mui/material'
 import SynapseSearchResultsCard from './SynapseSearchResultsCard'
 import SearchIcon from '@mui/icons-material/Search'
 import FilterAltOutlinedIcon from '@mui/icons-material/FilterAltOutlined'
-import React, { useState } from 'react'
+import React, { useState, useEffect } from 'react'
 import { useSearchInfinite } from '@/synapse-queries/search/useSearch'
 import { SearchQuery } from '@sage-bionetworks/synapse-types'
-import { SearchResults } from '@sage-bionetworks/synapse-types'
 
 export type SynapseSearchPageResultsProps = {
-  results: SearchResults
+  query?: string
+  setQuery?: (newQuery: string) => void
```

Comment:
This page will have to support more than just a string query, it will need to accept the [SearchQuery](https://rest-docs.synapse.org/rest/org/sagebionetworks/repo/model/search/query/SearchQuery.html) object, since we'll want to encode all of the other search options we expose in the URLSearchParam

```suggestion
  query?: SearchQuery
  setQuery?: (newQuery: SearchQuery) => void
```